tests/optimizers/bohb_test.py

```diff
@@ -250,3 +249,81 @@ def test_bohb_with_none_min_samples_in_model():
         min_samples_in_model=None,
     )
     assert opt.config_sampler.min_samples_in_model == 3
+
+
+def test_minimize_quadratic():
+    paramspace = ps.ParameterSpace()
+    paramspace.add(ps.ContinuousParameter("p1", (-2.0, 2.0)))
+    opt = BOHB(
+        paramspace,
+        Objective("loss", False),
+        min_fidelity=1.0,
+        max_fidelity=9.0,
+        num_iterations=10,
```

Comment:
Here is my proposal to make  this test reliable (without seeding).
I would restrict BOHB to one iteration here.